examples/src/main/java/com/yugabyte/examples/TopologyAwareLoadBalanceExample.java

```diff
@@ -16,18 +16,23 @@ public static void main(String[] args) {
     int argsLen = args.length;
     Boolean verbose = argsLen > 0 ? args[0].equals("1") : Boolean.FALSE;
     Boolean interactive = argsLen > 1 ? args[1].equals("1") : Boolean.FALSE;
+    debugLogging = argsLen > 2 ? args[2].equals("1") : Boolean.FALSE;
     // Since it is just a demo app. Using some smaller values so that it can run
     // on laptop
     String numConnections = "6";
     String controlHost = "127.0.0.1";
     String controlPort = "5433";
 
     controlUrl = "jdbc:postgresql://" + controlHost
-      + ":" + controlPort + "/yugabyte?user=yugabyte&password=yugabyte&load-balance=true&topology-keys=region1.zone1";
-
+      + ":" + controlPort + "/yugabyte?user=yugabyte&password=yugabyte&load-balance=true&topology-keys=cloud1.region1.zone1";
```

Comment:
This assignment to controlUrl can be enclosed in else block of the if block below.